docs/app-integrations/google-calendar.mdx

```diff
@@ -26,11 +26,51 @@ CREATE DATABASE my_calendar
 WITH
     ENGINE = 'google_calendar',
     PARAMETERS = {
-        'credentials': '\path-to-your-file\credentials.json'
+        'credentials_file': '\path-to-your-file\credentials.json'
     }; 
 ```
+<Tip>
+You need a Google account in order to use this integration. Here is how to get the credentials file:
+
+1. Create a Google Cloud Platform (GCP) Project:
+
+    1.1 Go to the GCP Console (https://console.cloud.google.com/).
+    
+    1.2 If you haven't created a project before, you'll be prompted to do so now.
+    
+    1.3 Give your new project a name.
+    
+    1.4 Click `Create` to create the new project.
+
+2. Enable the Google Calendar API:
+
+    2.1 In the GCP Console, select your project.
+
+    2.2 Navigate to `APIs & Services` > `Library`.
```

Comment:
Actually, that's the tab name:
2.2 Navigate to `APIs & Services` > `Enabled APIs & services`.